packages/web-components/src/components/va-file-input/va-file-input-upgrader.ts

```diff
@@ -0,0 +1,609 @@
+/* eslint-disable i18next/no-literal-string */
+import { Sanitizer, selectOrMatches } from '../../utils/utils';
+
+const PREFIX = 'usa';
+const DROPZONE_CLASS = `${PREFIX}-file-input`;
+const DROPZONE = `.${DROPZONE_CLASS}`;
+const INPUT_CLASS = `${PREFIX}-file-input__input`;
+const TARGET_CLASS = `${PREFIX}-file-input__target`;
+const INPUT = `.${INPUT_CLASS}`;
+const BOX_CLASS = `${PREFIX}-file-input__box`;
+const INSTRUCTIONS_CLASS = `${PREFIX}-file-input__instructions`;
+const PREVIEW_CLASS = `${PREFIX}-file-input__preview`;
+const PREVIEW_HEADING_CLASS = `${PREFIX}-file-input__preview-heading`;
+const DISABLED_CLASS = `${PREFIX}-file-input--disabled`;
+const CHOOSE_CLASS = `${PREFIX}-file-input__choose`;
+const ACCEPTED_FILE_MESSAGE_CLASS = `${PREFIX}-file-input__accepted-files-message`;
+const DRAG_TEXT_CLASS = `${PREFIX}-file-input__drag-text`;
+const DRAG_CLASS = `${PREFIX}-file-input--drag`;
+const LOADING_CLASS = 'is-loading';
+const INVALID_FILE_CLASS = 'has-invalid-file';
+const GENERIC_PREVIEW_CLASS_NAME = `${PREFIX}-file-input__preview-image`;
+const GENERIC_PREVIEW_CLASS = `${GENERIC_PREVIEW_CLASS_NAME}--generic`;
+const PDF_PREVIEW_CLASS = `${GENERIC_PREVIEW_CLASS_NAME}--pdf`;
+const WORD_PREVIEW_CLASS = `${GENERIC_PREVIEW_CLASS_NAME}--word`;
+const VIDEO_PREVIEW_CLASS = `${GENERIC_PREVIEW_CLASS_NAME}--video`;
+const EXCEL_PREVIEW_CLASS = `${GENERIC_PREVIEW_CLASS_NAME}--excel`;
+const SR_ONLY_CLASS = `sr-only`;
```

Comment:
This was changed from:

```
const SR_ONLY_CLASS = `${PREFIX}-sr-only`;
```

To:
```
const SR_ONLY_CLASS = `sr-only`;
```
If you import the usa-sr-only module into the stylesheet, I don't think you need to make that change.

```
@use 'uswds-helpers/src/styles/usa-sr-only';
```

For example: https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-text-input/va-text-input.scss#L3

Reply:
@Andrew565 I think you missed updating this `sr-only` Formation class to the USWDS `usa-sr-only`

https://github.com/department-of-veterans-affairs/component-library/blob/549701650399c9496c72af0399db41fa3ea4b428/packages/web-components/src/components/va-file-input/va-file-input.tsx#L188C17-L188C66

```html
  <span id="input-error-message" role="alert">
    {error && (
      <Fragment>
        <span class="sr-only">{i18next.t('error')}</span> <-------
        <span class="usa-error-message">{error}</span>
      </Fragment>
    )}
  </span>
```